src/main/java/frc/robot/RobotContainer.java

```diff
@@ -39,12 +46,19 @@ public class RobotContainer {
 
   private final RobotStateManager robotStateManager = new RobotStateManager();
 
+  private SendableChooser<Command> autoChooser;
+  private ShuffleboardTab configTab = Shuffleboard.getTab("Config");
+
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
     dynamicRobotConfig = new DynamicRobotConfig();
     drivetrain = dynamicRobotConfig.getTunerConstants().drivetrain;
     // Configure the trigger bindings
     configureBindings();
+    registerCommands();
+    autoChooser = AutoBuilder.buildAutoChooser();
+    configTab.add("Auton Selection", autoChooser);
```

Comment:
Can we add layout dimensions to this so its easy to see what is selected in Shuffleboard